angularjs-1.5-basics/src/app/binding-page/example.component.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
```

Comment:
__[jshint]__
* Use the function form of "use strict".

<!-- Comment by [SideCI](https://sideci.com) -->